src/store/index.ts

```diff
@@ -1,18 +1,17 @@
-// store.ts
-import { configureStore } from '@reduxjs/toolkit';
+import { configureStore, combineReducers } from '@reduxjs/toolkit';
 import createSagaMiddleware from 'redux-saga';
 import rootReducer from '../reducers/index';
```

Comment:
Inside a reducers folder, I would create a specific file for each reducers and index.ts for re-exports. It makes the structure more extendable, for instance, your import will have the next view:

```js 
import { todosReducer } from '../reducers'
```

and:

```js
reducer: combineReducers({
    todos: todosReducer    
}),
```